framework/components/ecl-files/ecl-files.twig

```diff
@@ -1,112 +1,144 @@
+          {% for translation in translations %}
+        <li class="ecl-file__translations-item">
+          <div class="ecl-file__translations-metadata">
+            <span class="ecl-file__translations-title">{{ translation.title }}</span>
+            <div class="ecl-file__translations-info">({{ translation.meta }})</div>
+          </div>
+          {% include '@ec-europa/ecl-buttons' with {
+          'label': buttonLabel ~ '<span class="ecl-sr-only">(' ~ translation.meta ~ ')',
+          'modifier': 'secondary',
+          'extraClass': 'ecl-file__translations-download',
+          'extraAttributes': [
+          { 'name': 'type', 'value': 'button' },
+          ]
+          } %}
+        </li>
+          {% endfor %}
+        {% endif %}
+      </ul>
     </div>
   </div>
-  <a href="#" class="btn btn-default file__btn">
-    Download<span class="ecl-sr-only">(392.23 KB - PDF - 2 pages)</span>
-  </a>
-</div>
-
-<div class="file__translations" id="file-translations">
-  <a href="#file" class="file__translations-control" title="Click to see translations" data-toggle="collapse" data-parent="#file-translations" aria-expanded="false" aria-controls="file">Available languages (3)</a>
-  <ul id="file" class="file__translations-list collapse in"><!-- Class .in is just for the style guide -->
-    <li class="file__translations-item">
-      <div class="file file--bare">
-        <div class="file__metadata">
-          <span class="file__title">Name language</span>
-          <div class="file__info">(392.23 KB - PDF - 2 pages)</div>
-        </div>
-        <a href="#" class="btn file__btn btn--condensed is-internal">
-          Download<span class="ecl-sr-only">(392.23 KB - PDF - 2 pages)</span>
-        </a>
-      </div>
-    </li>
-    <li class="file__translations-item">
-      <div class="file file--bare">
-        <div class="file__metadata">
-          <span class="file__title">Name language</span>
-          <div class="file__info">(392.23 KB - PDF - 2 pages)</div>
+</section>
+
+  {% elseif variant == "links" %}
+
+<!-- Links -->
+<section class="ecl-file ecl-file--link">
+  {% if links is defined %}
+    {% for link in links %}
+      <a
+        href="{{ link.href }}"
+        class="ecl-link ecl-file__link {% if link.external is defined and link.external %}ecl-link--external{% endif %}">
+        <span class="ecl-icon ecl-icon--{{ link.icon }} ecl-file__icon"></span>
+        <span class="ecl-file__title">{{ link.title }}</span>
+        {% if link.type is defined %}
+        <span class="ecl-file__type">{{ link.type }}</span>
+        {% endif %}
+      </a>
+    {% endfor %}
+  {% endif %}
+</section>
+
+  {% elseif variant == "images" %}
+
+<!-- Images -->
+<section class="ecl-file ecl-file--image">
+  <img class="ecl-file__image" src="{{ src }}" alt="{{ alt }}">
+  <div class="ecl-file__caption">{{ caption }}</div>
+</section>
+
+  {% elseif variant == "Videos" %}
+
+<!-- Videos -->
+<section class="ecl-file ecl-file--video">
```

Comment:
It would be much appreciated to keep the aspect ratio. See https://css-tricks.com/snippets/sass/maintain-aspect-ratio-mixin/ :)